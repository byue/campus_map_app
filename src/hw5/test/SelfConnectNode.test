# Tests that a graph can be created, and a node
# can be added to the graph with an edge to itself. 

# Create a graph
CreateGraph graph1

# Add a node
AddNode graph1 n1

# Connect node to itself
AddEdge graph1 n1 n1 a

# list nodes of graph
ListNodes graph1

# list children of node
ListChildren graph1 n1