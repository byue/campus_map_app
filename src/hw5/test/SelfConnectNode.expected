# Tests that a graph can be created, and a node
# can be added to the graph with an edge to itself. 

# Create a graph
created graph graph1

# Add a node
added node n1 to graph1

# Connect node to itself
added edge a from n1 to n1 in graph1

# list nodes of graph
graph1 contains: n1

# list children of node
the children of n1 in graph1 are: n1(a)